### ISSUE TITLE
Governance: can/should peers modify milestones, projects, and/or labels?

### ISSUE THREAD

**Issue by vinyldarkscratch** [mdn/browser-compat-data#4417]
This is something that has been itching the back of my mind, and I thought I'd bring it up for review.  As I'm going through pull requests and issues, sometimes I find myself tempted to add something to a milestone or the project board(s).  I've been commonly adding issues and PRs regarding non-data issues to the corresponding board, and I would assume that's a part of my role as a peer, however something like milestones I would assume is more for the repository owners to manage.  Am I correct on this?

Bottom line, I would like to see these details added to the governance doc, helping add a little more clarity to each group's permissions.

**Comment by Elchi3** [comment]
The current milestones are for the MDN staff team's sprints (you could create separate, own milestones, but that didn't happen so far and I'm unsure how practical it would be). The employed Mozillians work in agile sprints and workload is assigned and planned accordingly. If peers/owners aren't part of these sprints and/or sprint planning conversations, they shouldn't touch milestones. If you want to be part of a sprint as a volunteer and want items assigned to you on during a sprint iteration, let me know and we will have to see how get you more involved. Note that what gets put into sprints needs to be aligned with approved yearly goals etc. So, these conversations also happen with product management and are often juggled against all the other things MDN wants to get done in a time period.

The BCD project boards aren't really used by anyone but me. Feel free to update them, I don't mind too much :)

Does this answer the questions?

**Comment by jpmedley** [comment]
FWIW, I wouldn't feel remotely comfortable touching an MDN milestone.

**Comment by vinyldarkscratch** [comment]
That answers my questions, thank you!  Regarding new milestones, personally I feel that peers should only create new milestones upon owner approval, at least from a management standpoint.  I can

**Comment by vinyldarkscratch** [comment]
(Sometimes that “Close and comment” button gets on my nerves, especially from my iPhone...rrrrr...  😫)

That answers my questions, thank you!  Regarding new milestones, personally I feel that peers should only create new milestones upon owner approval, at least from a management standpoint.  I can see non-sprint milestones possibly being used for tracking the process of bulk changes, including a date when a migration would occur (like the removal of a browser — linking to #4469 which talks more about the flow of bulk changes).

As for assisting with sprints, I’m definitely interested in aiding however I can, yeah!  Sign me up.  😄

Finally, with the project boards, good to know!  I find it pretty handy myself to keep track of all that’s going around within this repository.  I’ll be sure to keep it up to date without messing around too much.  😉
